mne/defaults.py

```diff
@@ -83,9 +87,7 @@ def _handle_default(k, v=None):
     """Avoid dicts as default keyword arguments.
 
     Use this function instead to resolve default dict values. Example usage::
-
```

Comment:
This PR doesn't relate to `_handle_default`, so no need to change the doctoring here.

Reply:
I am failing tests because of this function